rosie.py

```diff
@@ -1,4 +1,20 @@
+"""Rosie.
+
+Usage:
+    python rosie.py run (chamber_of_deputies | federal_senate)
+    python rosie.py run (chamber_of_deputies | federal_senate) [--years 2017,2016]
+    python rosie.py run (chamber_of_deputies | federal_senate) [--path <path to output directory>]
+    python rosie.py run (chamber_of_deputies | federal_senate) [--path <path to output directory>] [--years 2017,2016]
+    python rosie.py test [chamber_of_deputies | federal_senate]
+
+Options:
+    --years   runs rosie wuth years filter
+    --path    runs rosie with specific directory
+
+"""
+
 from sys import argv, exit
+from docopt import docopt
```

Comment:
PEP8 suggests a blank line between native modules (`sys`) and third-party modules (`docopt`).